docs/source/usage_guides/big_modeling.md

```diff
@@ -64,7 +64,7 @@ Now that the model is fully dispatched, you can perform inference.
 
 ```py
 input = torch.randn(2,3)
-input = input.to("cuda")
+input = input.to(model.device.type)
```

Comment:
I'm afraid the `model` does not always have a `device` member.

Reply:
also when the model is loaded with `load_checkpoint_and_dispatch`? I think when we consider the context, model should be a HF ModelClass, which will have device as an attribute. But pls correct me if I am wrong.

Reply:
We don't, esp as the prior sentence considers a regular pytorch model.

We can instead do one of two options:

1. Bring in the `PartialState` for this tutorial
2. Go a bit easier and use the model itself

(More a fan of 2 here)

```suggestion
device_type = next(iter(model.parameters())).device.type
input = input.to(device_type)
```